equinix/helpers_device.go

```diff
@@ -140,30 +123,15 @@ func getNetworkRank(family int, public bool) int {
 	return 3
 }
 
-func getPortsMetalGo(ps []metalv1.Port) []map[string]interface{} {
-	ret := make([]map[string]interface{}, 0, 1)
-	for _, p := range ps {
-		port := map[string]interface{}{
-			"name":   p.Name,
-			"id":     p.Id,
-			"type":   p.Type,
-			"mac":    p.Data.Mac,
-			"bonded": p.Data.Bonded,
-		}
-		ret = append(ret, port)
-	}
-	return ret
-}
-
-func getPorts(ps []packngo.Port) []map[string]interface{} {
+func getPorts(ps []metalv1.Port) []map[string]interface{} {
 	ret := make([]map[string]interface{}, 0, 1)
 	for _, p := range ps {
 		port := map[string]interface{}{
-			"name":   p.Name,
-			"id":     p.ID,
-			"type":   p.Type,
-			"mac":    p.Data.MAC,
-			"bonded": p.Data.Bonded,
+			"name":   p.GetName(),
```

Comment:
As with `getNetworkInfo`, this change replaces the innards of `getPorts` with those of `getPortsMetalGo` while also updating the function to use `Get...()` helpers, since those helpers more closely match the behavior of the `packngo` implementation.